Makefile.core.mk

```diff
@@ -1,21 +1,33 @@
-export GO111MODULE ?= on
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-lint:
-	@scripts/run_golangci.sh
-	@scripts/check_license.sh
-	@bazel run //:buildifier -- -showlog -mode=check $(git ls-files| grep -e BUILD -e WORKSPACE | grep -v vendor)
+lint: lint-go
+
+test:
+	@go test -race ./...
 
 .PHONY: testgrid
 testgrid:
-	configurator --prow-config prow/config.yaml --prow-job-config prow/cluster/jobs --output-yaml --yaml testgrid/default.yaml --oneshot --output testgrid/istio-generated.yaml
+	@configurator --prow-config prow/config.yaml --prow-job-config prow/cluster/jobs --output-yaml --yaml testgrid/default.yaml --oneshot --output testgrid/istio-generated.yaml
```

Comment:
this isn't in the container right?

Reply:
Correct. What is configurator and where does it come from?

Reply:
I'm guessing this is the source for this tool:

https://github.com/kubernetes/test-infra/tree/master/testgrid/cmd/configurator

Reply:
I've updated this to build the tool on the fly, but it's not quite working yet. Not clear why...

Reply:
Works now. We could leave it this way, or we could put configurator into the build container. Not sure if its worth it. Thoughts?